### DISCUSSION TITLE
Default tags from dagit UI in partition backfill

### DISCUSSION THREAD

**Discussion by dridk** [dagster-io/dagster#22808] [Q&A] [answered]
Hi, 

I would like to know if it is possible to set a default run tags in dagit backfill panel  ? 
Actually, I need to set manually from dagit each materialization with a specific tag ( used for concurrent run ) .  I would like to set this run tag programmatically from asset definition. 

![image001 (2).png](https://github.com/dagster-io/dagster/assets/1911063/72b62170-7407-4c85-8283-a6f5ba2e6a93)

**Comment by garethbrickman** [comment, marked as answer]
It is possible to set the backfill run's tags when using the Dagster CLI, as shown in: https://github.com/dagster-io/dagster/issues/11585

[Looking at the code for when the CLI does this](https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/dagster/_cli/job.py#L746), I don't think this can be done via an asset definition because they are run tags.
